students/Jaidjen/lesson03/Mailroom1.py

```diff
@@ -0,0 +1,73 @@
+import sys  # imports go at the top of the file
+
+
+Mail  =[('James Smith',[2000.25, 500, 27.50]),
```

Comment:
this should be lower case snake_case > `mail`

I would also recommend naming this `donors` to more fit what this data holds